config/pmd/ruleSets.xml

```diff
@@ -29,7 +29,7 @@
   </rule>
   <rule ref="rulesets/java/j2ee.xml"/>
   <!--<rule ref="rulesets/java/javabeans.xml"/>-->
-  <rule ref="rulesets/java/junit.xml"/>
+  <!--<rule ref="rulesets/java/junit.xml"/>-->
```

Comment:
Rules from this set are useless: http://pmd.sourceforge.net/pmd-4.3.0/rules/junit.html